core/mingw/test/files/SmokeFileTestWindows.kt

```diff
@@ -20,4 +18,13 @@ class SmokeFileTestWindows {
         assertFalse(Path("bla\\bla\\bla").isAbsolute)
         assertTrue(Path("\\\\server\\share").isAbsolute)
     }
+
+    @Test
+    fun getParent() {
+        assertNull(Path("C:\\").parent)
```

Comment:
Shouldn't we add a mixed delimiters test, like `Path("C:\\a/b")`?

Reply:
The last case in this test is about mixed delimiters